client/mobileClient/MobileClient/Station.js

```diff
@@ -8,40 +8,157 @@ import React, {
   TouchableHighlight,
   View,
 } from 'react-native';
+import Loading from './Loading';
 
 var audio = require('react-native').NativeModules.RNAudioPlayerURL;
 
+var FAKE_STATION_DATUM = {
+  heartCount: 1000,
+  image: '',
+  listenerCount: 20,
+  desc: 'Sick beets. Not a typo. Here is some additional text to experiement with how text flows.'
+}
+
+var REQUEST_URL_ONE = "";
+
 class Station extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoading: true,
+      heartCount: 0,
+      image: '',
+      listenerCount: 0,
+      desc: ''
+    }
+  }
 
   playStation() {
     audio.play();
   }
 
   componentDidMount() {
-    audio.initWithURL(this.props.station.stationInfo.uri);
+    //this.fetchData();
+    var stationDetail = FAKE_STATION_DATUM;
+    console.log(this.props.station.id);
+    this.setState({
+      isLoading: false,
+      heartCount: stationDetail.heartCount,
+      image: "https://i.imgur.com/03t6Yun.jpg",
+      listenerCount: stationDetail.listenerCount,
+      desc: stationDetail.desc
+    });
+    audio.initWithURL(this.props.station.uri);
+    audio.play();
+  }
+
+  componentWillUnmount() {
+    audio.pause();
+  }
+
+  fetchData() {
+    fetch(REQUEST_URL_ONE)
+      .then((response) => response.json())
+      .then((responseData) => {
+        this.setState({
+          isLoading: false,
+          heartCount: responseData.heartCount,
+          image: responseData.image,
+          listenerCount: responseData.listenerCount,
+          desc: responseData.desc
+        });
+      })
+      .done();
+  }
+
+  upheart() {
+    var POST_HEART = "http://10.6.32.108:8000/" + this.props.station.id + "/heart";
+    fetch(POST_HEART, {
+      method: 'POST',
+      body: JSON.stringify({
+        id: this.props.station.id
+      })
+    })
+    .then((response) => response.json())
+    .then((responseData) => {
+      this.setState({
+        heartCount: responseData.heartCount.length > 5 ? ">9,999" : responseData.heartCount
```

Comment:
I think we can come up with a technical solution here. Let's revisit this issue,